data/Music/Final Fantasy Series/Final Fantasy X/Battle.meta

```diff
@@ -1,3 +1,3 @@
 Final Fantasy X - Battle
```

Comment:
Why are you replacing this song instead of adding a second version? What's changed?

Reply:
they fixed their own track :) so thats totally fine ^^